### PR TITLE
[CodingStyle] Failing fixture for MakeInheritedMethodVisibilitySameAsParentRector prepend public to protected on static

### LINKED ISSUE #4941: MakeInheritedMethodVisibilitySameAsParentRector and static methods
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version |dev-master              |
| Installed as   | composer dependency             |

## Minimal PHP Code Causing Issue

https://getrector.org/demo/c870185a-b4e7-4f63-abdf-e97798a656ba#demo_result

## Expected Behaviour

Refactored code should NOT have "public" keyword.

### PR DESCRIPTION
**Pull request by samsonasik**
Ref #4941

### CODE REVIEW

**Comment by samsonasik** [comment]
fixed.

File: rules/generic/tests/Rector/ClassMethod/ChangeMethodVisibilityRector/Fixture/fixture2.php.inc [hunk 1]
```diff
@@ -17,7 +17,7 @@ use Rector\Generic\Tests\Rector\ClassMethod\ChangeMethodVisibilityRector\Source\
 
 class ClassThatInherits2 extends ParentObject
 {
-    public function toBePublicStaticMethod() {
+    public static function toBePublicStaticMethod() {
```
Comment (samsonasik) [inline rules/generic/tests/Rector/ClassMethod/ChangeMethodVisibilityRector/Fixture/fixture2.php.inc:20]: it change `ChangeMethodVisibilityRector` tests as `ParentObject::toBePublicStaticMethod()` is static too.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Review by TomasVotruba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**